front-end/src/shared/SideBarNav.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
```

Comment:
You can ignore this as well. Part of @PaulNewling's work for the view profile page but needed it to confirm that Login worked